examples/dummy_plugin/dummy_plugin/api/serializers.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework import serializers
+
+from nautobot.core.api import ValidatedModelSerializer
+
+from dummy_plugin.models import DummyModel
+
+
+class DummySerializer(ValidatedModelSerializer):
+    url = serializers.HyperlinkedIdentityField(view_name="plugins-api:dummy_plugin-api:dummymodel-detail")
+
+    class Meta:
+        model = DummyModel
+        fields = ("url", "id", "name", "number")
```

Comment:
Not blocking for this PR but at some point for completeness we may want to add a NestedDummySerializer as well.